backend/migrations/20220822091356_introduce_endpoints.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE endpoint
+(
```

Comment:
Would recommend adding an `is_public` field to allow non authentified users to reach it. Otherwise, the path _prefix need to follow the path structure and the ownership will be determined based on it. Now we also need to add granular permissions to allow for additional ppl to read/write even if they are not owner